pyproject.toml

```
@@ -83,10 +83,13 @@ dependencies = [
 ]
 
 requires-python = ">=3.8"
-license = { file = "LICENSE" }
 dynamic = ["version"]
 
+[metadata]
+license = { text = "BSD 3-Clause License" }
+
 [project.urls]
+"Homepage" = "https://sourmash.readthedocs.io/"
```

```suggestion
"Homepage" = "https://sourmash.bio/"
```

It is just a redirect, but why not? =]